android/src/main/java/com/swmansion/rnexecutorch/LLM.kt

```diff
@@ -148,10 +149,6 @@ class LLM(reactContext: ReactApplicationContext) :
     llamaModule!!.stop()
   }
 
-  override fun deleteModule() {
```

Comment:
Why are we getting rid of this?

Reply:
I noticed that it doesn't really change memory usage, but let's leave it for now as it's not breaking anything